_posts/2018-06-18-12-02-WebApiV7_newDefaultSettings.md

```diff
@@ -0,0 +1,285 @@
+---
+layout: post
+title: "12.1 WebApi 7.0 Default Setting Updates"
+description: "Desgin document for updated default settings for WebApi 7.0"
+category: "12. Design"
+---
+        
+# WebApi Default Enable Unqualified Operations and Case-insensitive Uri #
+
+
+Overview
+========
+
+OData Layer
+-----------
+
+OData libraries 7.x (current version 7.4.4) contains updates to improve
```

Comment:
I think the doc will be outdated as soon as we release 7.5.0. Should we just leave it at 7.x? Or say 7.5.0+?